### ISSUE TITLE
Running make -f from the directory in which the makefile resides

### ISSUE THREAD

**Issue by masteranza** [microsoft/vscode-makefile-tools#80]
What is the proper way to make "make -f" run from the directory set in makefile.makefilepath ?
I need this because my makefile includes other makefiles from the same directory.
I'm currently using a hack, that is I modify the makefile.makepath to "cd dir; make"...

**Comment by andreeis** [comment]
Yes, -C in makeArgs is one way to do this.
I was experimenting at some point with the extension adding -C automatically for makefilePath but I found some repos broken with this approach so I'm still brainstorming about that. Use -C in makeArgs for now.

**Comment by KrestenHelstrup** [comment]
Might be a stupid questions - But I have exactly the same problem and I don't know where I find this "makeArgs", could you help a bit.
Thanks

**Comment by andreeis** [comment]
Of course. Here's an example. After saving settings.json, you can go to the Makefile Tools UI side panel (left, bottom), click on the 'edit pen' on the line with "Configuration" and select "MyConfiguration". This will automatically trigger a configure (unless you set makefile.configureOnCommand to false) or if it doesn't run the configure command from the pallette.

If you defined makefile.makefilePath already to your main makefile, don't give -f to makeArgs.

    "makefile.configurations": [
        {
            "name": "MyConfiguration",
            "makeArgs": ["-C /my/folder", "-f /my/folder/myMakefile"]
        }
    ],

**Comment by KrestenHelstrup** [comment]
@andreeis Works perfect - Thank you.
